### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by timurvafin**
* Remove Upgrade outdated and wrong section, need to get back
* Remove Foundation relted sections, should be part of Wiki or somethings else
* Rewrite install sections

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,57 +64,57 @@ It's based on Rails 4 and Ruby 2.1.5.
...
-Clone application as new project with original repository named "rails-base"
-
-    git clone --depth 1 git://github.com/fs/rails-base.git --origin rails-base [MY-NEW-PROJECT]
-
-**Note: we use depth parameter here in order not to copy the history of changes in base project**
+### Prepare dependencies
 
-Create your new repo on GitHub and push master into it.
-Make sure master branch is tracking origin repo.
+Application based on Ruby 2.1.5 and PostgreSQL, some gems have native extensions.
```
Comment (vast) [inline README.md]: we already wrote about ruby version earlier:

> It's based on Rails 4 and Ruby 2.1.5

File: README.md [hunk 2]
```diff
@@ -67,57 +64,57 @@ It's based on Rails 4 and Ruby 2.1.5.
...
+You should have GCC installed on you development machine.
```
Comment (vast) [inline README.md]: youR

File: README.md [hunk 3]
```diff
@@ -67,57 +64,57 @@ It's based on Rails 4 and Ruby 2.1.5.
...
 
-    git remote add origin git@github.com:[MY-GITHUB-ACCOUNT]/[MY-NEW-PROJECT].git
-    git push -u origin master
+* `brew install qt` - to run spec with [Capybara Webkit](https://github.com/thoughtbot/capybara-webkit)
```
Comment (vast) [inline README.md]: specS

File: README.md [hunk 4]
```diff
@@ -67,57 +64,57 @@ It's based on Rails 4 and Ruby 2.1.5.
...
-    bin/ci
+3. Run setup script
 
-Run app
+  ```bash
+  bin/setup
+  ```
 
-    bin/server
+4. Make sure all test are green
```
Comment (vast) [inline README.md]: tests

Comment (vast) [inline README.md]: why do I have to run all tests before starting the application?

File: README.md [hunk 5]
```diff
@@ -67,57 +64,57 @@ It's based on Rails 4 and Ruby 2.1.5.
...
+  bin/server
+  ```
 
-You can fetch latest changes from rails-base repo and merge or cherry-pick commits
+6. Update README
 
-    git fetch rails-base
-    git flow feature start rails-base-update
-    git merge rails-base/master
+  Do not forget to update application README.md file with detailed information baed on the
```
Comment (vast) [inline README.md]: baSed

File: README.md [hunk 6]
```diff
@@ -67,57 +64,57 @@ It's based on Rails 4 and Ruby 2.1.5.
...
+  existing tempalte.
```
Comment (vast) [inline README.md]: tempLAte

File: README.md (hunk 4 above)
Comment (timurvafin) [inline README.md]: to make sure all deps are satisfied and applications works correctly before making changes.

**Comment by vast** [comment]
ok

**Pull request merged**